pulp_container/app/models.py

```diff
@@ -138,10 +140,17 @@ def init_annotations(self, manifest_data=None):
 
     def init_labels(self):
         if self.config_blob:
-            config_artifact = self.config_blob._artifacts.get()
-
-            config_data, _ = get_content_data(config_artifact)
-            self.labels = config_data.get("config", {}).get("Labels") or {}
+            try:
```

Comment:
you could get rid of try/except block if you just simply check for `if self.config_blob.data`